playground/backend-integration/tailwind.config.js

```diff
@@ -1,5 +1,4 @@
 module.exports = {
-  mode: 'jit',
   purge: [__dirname + '/frontend/**/*.{css,html,ts,js}'],
```

Comment:
```text
warn - The `purge`/`content` options have changed in Tailwind CSS v3.0.    
warn - Update your configuration file to eliminate this warning.
warn - https://tailwindcss.com/docs/upgrade-guide#configure-content-sources

warn - The `darkMode` option in your Tailwind CSS configuration is set to `false`, which now behaves the same as `media`.
warn - Change `darkMode` to `media` or remove it entirely.
warn - https://tailwindcss.com/docs/upgrade-guide#remove-dark-mode-configuration 
```
It seems some options needs to be updated.

Reply:
Thanks yeah did this upgrade a bit too quickly 😅